components/cam/src/physics/crm/rrtmgp/radiation.F90

```diff
@@ -276,7 +286,8 @@ subroutine radiation_readnl(nlfile, dtime_in)
              '  Use average zenith angle:                           ',l5/, &
              '  Output spectrally resolved fluxes:                  ',l5/, &
              '  Do aerosol radiative calculations:                  ',l5/, &
-             '  Fixed solar consant (disabled with -1):             ',f10.4/ )
+             '  Fixed solar consant (disabled with -1):             ',f10.4/, &
+             '  Enable limiter warnings:                            ',l5/ )
```

Comment:
This should say "Enable temperature warnings" since you changed the name of the variable.

Reply:
Good call. Done.